src/requests/request.rs

```diff
@@ -51,15 +52,15 @@ pub trait Request: HasPayload {
     /// ```
     fn send(self) -> Self::Send;
 
-    /// Send the request.
+    /// Send this request by reference.
     ///
     /// This method is analogous to [`send`](Request::send), but it doesn't take
     /// the ownership of `self`. This allows to send the same (or slightly
     /// different) requests over and over.
     ///
-    /// _Also_ it is expected that calling this method is better than just
-    /// `clone`ing the requests. (because instead of copying all the data
-    /// and then serializing it, this method should just serialize the data)
+    /// Also, it is expected that calling this method is better than just
+    /// cloning requests. (Because instead of copying all the data
+    /// and then serializing it, this method should just serialize the data.)
```

Comment:
I haven't seen dots at the end of a sentence in braces before, are you sure it's mandatory?

Reply:
Yes, see https://www.gutenberg.org/files/37134/37134-h/37134-h.htm#Page_33.

Reply:
Feels strange to me, but ok, thanks for the link.